lib/sycamore/sycamore/query/execution/operations.py

```diff
@@ -52,14 +58,13 @@ def math_operation(val1: int, val2: int, operator: str) -> Union[int, float]:
 @context_params
 def summarize_data(
```

Comment:
Can we simplify this? It feels weird to have a summarize class and a whole ton of parameters in the call.